components/user-details.vue

```diff
@@ -209,6 +223,22 @@
         return (this.user.affiliation === 'BUSINESS' && this.user.companyName != null)
       },
 
+      isRepeatCustomer () {
+        let count = 0
+        this.orders.forEach(element => {
+          if (this.orderId) {
+            if (element.status === 'shipped_complete' && this.orderId !== element.id) count++
+          } else {
+            if (element.status === 'shipped_complete') count++
+          }
+        })
+        if (count > 0) {
+          return true
+        } else {
+          return false
+        }
+      },
```

Comment:
Why is this function so complex? Repeat customer shouldn't have to take into considering the current order. It should just be if they have _any_ order that is shipped, no?

```suggestion
      isRepeatCustomer () {
        const shippedOrders = this.orders
          .filter((o) => ['shipped_complete'].includes(o.status))
        
        return (shippedOrders.length >= 2)
      }
```